nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -591,6 +591,26 @@
      The defaults from fontconfig are sufficient.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Using <option>fonts.enableDefaultFonts</option> adds a default emoji font <literal>noto-fonts-emoji</literal>.
+     <itemizedlist>
+      <para>Users of the following options will have this enabled by default:</para>
+      <listitem>
+       <para><option>services.xserver.enable</option></para>
```

Comment:
Is not this option enabled by GNOME and other DEs? Maybe point that out.

Reply:
Yep, will note that all DE modules enable this.

Reply:
From a quick rg no, this has to be enabled manually along with the desktopManager.